pkg/volume/util/operationexecutor/operation_generator.go

```diff
@@ -749,11 +750,23 @@ func (og *operationGenerator) GenerateMountVolumeFunc(
 		}
 
 		// Execute mount
-		mountErr := volumeMounter.SetUp(volume.MounterArgs{
+		opExitStatus, mountErr := volumeMounter.SetUp(volume.MounterArgs{
```

Comment:
Now we introduce this new operation state, should we log the returned state?

Reply:
I am thinking that, while keeping all the underlying logic same, use special error types to indicate different state of failures. It will reduce the size of change and will allow us to keep volume plugin interface, more or less same. Thanks to @saad-ali for proposing this idea.

Reply:
fixed